charts/op-scim-bridge/values.yaml

```
@@ -213,6 +213,8 @@ redis:
                     values:
                       - op-scim-bridge
               topologyKey: "kubernetes.io/hostname"
+    persistence:
+      enabled : false
```

Little typo here that the linter already caught:

```suggestion
      enabled: false
```